assets/js/vue-apps/grant-filters.js

```diff
@@ -82,9 +91,14 @@ function init() {
 
             // Convert filters into URL-friendly state
             filtersToString: function() {
-                return Object.keys(this.filters)
+                let filterClone = Object.assign({}, this.filters);
```

Comment:
Maybe use `lodash/cloneDeep` or `lodash/clone` here?

Reply:
Reading this back this sentence is confusing. I mean something like this:

```js
const filterClone = cloneDeep(this.filters);
const cleanFilters = this.ignoreSort && filterClone.sort ?  omit(filterClone, 'sort') : filterClone;
```

Reply:
Good shout, give it a look now.